spec/reek/smells/irresponsible_module_spec.rb

```diff
@@ -61,6 +66,80 @@ class BadClass
     expect(src).to reek_of :IrresponsibleModule, name: 'Foo::Bar'
   end
 
+  it 'does not report modules used only as namespaces' do
```

Comment:
I don’t think this describes the spec, which seems to be about not reporting a documented class.